web/src/app/services/appid-authentication.service.ts

```diff
@@ -25,9 +25,9 @@ export class AuthenticationService {
     try {
     this.appid.init({
       // REPLACE with credentials from IBM CLOUD APPID Service
-      clientId: "f49ba5e4-e6ab-4930-9682-6f74d94d5a09",
+      clientId:  '<CLIENT-ID>',
```

Comment:
@wisdomkeeper thank you for reverting this change. You might also want to recreate your personal credentials as they are not in GitHub history.